### ISSUE TITLE
Alternative ports producing 503 error

### ISSUE THREAD

**Issue by steveh138** [jwilder/nginx-proxy#1143]
When using this as a front end to a grafana installation alternative ports are not working. This is with both the virtual_port environment variable being used and not being used (port 3000 is the only exposed port) it produces the same error. Here are the compose files below.
version: '3'

nginx-proxy compose file

services:
  nginx-proxy:
    image: 127.0.0.1:5000/nginx-stickyproxy .  <- nginx-proxy container image renamed
    volumes:
      - /var/run/docker.sock:/tmp/docker.sock:ro
    ports:
      - "80:80"
      - "443:443"
    networks:
      - net
networks:
  net:
    external: true

---------------------------------------
grafana compose file 

version: '3'

services:
  webui:
    image: 127.0.0.1:5000/grafana
    environment:
      VIRTUAL_HOST: ussl-grafana.usanainc.com
      GF_SERVER_ROOT_URL: http://ussl-grafana.usanainc.com
      GF_SECURITY_ADMIN_PASSWORD: usana123
    volumes:
      - /srv/docker_content/grafana:/var/lib/grafana
    networks:
      - net
networks:
  net:
    external: true

**Comment by steveh138** [comment]
It appears this is not really designed for warms. Unless the container happens to land on the same node as the nginx proxy it will not update the config. Could you please update your documentation to reflect that? This was not a listed limitation. At least I didn't notice it when trying to select a sticky proxy to use. Thank you

**Comment by CamoMacdonald** [comment]
Yeah, its currently not designed to work with swarm mode. However it can be customised to do so however it takes away some of the automation
